### PR TITLE
Basic mocked api json data being returned for client side implementation 

### PR DESCRIPTION
**Pull request by moranthomas**

### CODE REVIEW

File: routes/curations.js [hunk 1]
```diff
@@ -52,10 +52,18 @@ router.get(
...
   '/:type?/:provider?/:namespace?/:name?',
   asyncMiddleware(async (request, response) => {
-    const coordinates = utils.toEntityCoordinatesFromRequest(request)
-    return curationService.list(coordinates).then(result => response.status(200).send(result))
-  })
-)
+      const coordinates = utils.toEntityCoordinatesFromRequest(request)
+      curationService.list(coordinates).then(function(result){
+         /* HANDLE PROMISE #1 */
+         var text_file_name = "./test/fixtures/curations_dummy_data.json";
```
Comment (AlexWebYourmind) [inline routes/curations.js]: @moranthomas are we sure this is a var? is it not better to use an es6 const in this case.
I would recommend that we remain consistent with the overall codebase, so e.g. use let vs var, const where needed etc.

File: routes/curations.js [hunk 2]
```diff
@@ -90,4 +98,16 @@ function setup(service) {
   return router
 }
 
+//synchronous version
+function readTextFile(file) {
+    var fs=require('fs');
```
Comment (AlexWebYourmind) [inline routes/curations.js]: const should be used where value won't change.

File: routes/curations.js (hunk 1 above)
Comment (moranthomas) [reply routes/curations.js]: Thanks Alex. I agree. I have updated the code with your suggested edits and re-committed

File: routes/curations.js (hunk 2 above)
Comment (moranthomas) [reply routes/curations.js]: Thanks Alex. I agree. I have updated the code with your suggested edits and re-committed

File: routes/curations.js [hunk 3]
```diff
@@ -54,8 +54,41 @@ router.get(
...
+        console.log('url', url);
+
+        //trim and split the request url for processing
+        let trimmed = url.trim();
+        let myStringArray = trimmed.split("/");
+        let text_file_name = "./test/fixtures/dummy_data_curations_"
+
+        //iterate through the 'url array' and create a filename string from it
+        const arrayLength = myStringArray.length;
+        for (var i = 0; i < arrayLength; i++) {
```
Comment (AlexWebYourmind) [inline routes/curations.js]: @moranthomas we should really consider using let in place of var, especially inside for loops. No need to optimize this now tho.

**Comment by AlexWebYourmind** [comment]
@storrisi can you merge when you have the chance? Make sure to check out my review comments.

**Pull request merged**